hpobench/container/recipes/surrogates/Singularity.ParamnetBenchmark

```diff
@@ -0,0 +1,21 @@
+Bootstrap: docker
+From: python:3.7-slim
+
+%labels
+MAINTAINER muelleph@cs.uni-freiburg.de
+VERSION v0.0.1
+
+%post
+    apt update -y \
+    && apt install build-essential git wget -y
+
+    cd /home \
+    && git clone https://github.com/automl/HPOBench.git \
+    && cd HPOBench \
+    && pip install --upgrade .[paramnet] \
```

Comment:
Why --upgrade? We don't do this for other benchmarks?

Reply:
This is because we need an older scikit-learn version. Saving and restoring an TreeRegressor is only allowed until 0.25 I guess. There is a warning about it. Therefore, I put the new scikit-learn version in the requirements and to make sure it is installed, I've added the --upgrade parameter. 
Without it, pip is already happy if there is it already installed.

Reply:
Okay, so this is fine then.